src/components/pages/HomePage.tsx

```diff
@@ -0,0 +1,14 @@
+import { Navigate, useParams } from 'react-router-dom';
+import { NotFoundPage } from './NotFoundPage';
+
+export const HomePage = () => {
+  const { home } = useParams();
+
+  if (home === 'home') {
+    return <Navigate to="/" replace={true} />;
+  } else if (home) {
+    return <NotFoundPage />;
+  }
+
+  return <h1 className="title">Home Page</h1>;
```

Comment:
This is the expected behavior for a home page - to render some sort of home page content. If you want to keep the conditional logic above, consider making this the default return statement, rather than nested within an else clause.